static/js/settings.js

```diff
@@ -55,14 +55,70 @@ const SettingsScreen = {
     this.availableAddons = new Map();
     this.installedAddons = new Map();
     this.fetchAddonDeferred = null;
+
+    this.insertTitleElement(this.menu, 'Settings',
+                            '/optimized-images/settings-icon.png');
+    this.insertTitleElement(this.domainSettings, 'Domain',
+                            '/optimized-images/domain-icon.png');
+    this.insertTitleElement(this.userSettingsMain, 'Users',
+                            '/optimized-images/users-icon.png');
+    this.insertTitleElement(this.userSettingsEdit, 'Edit User',
+                            '/optimized-images/user.svg');
+    this.insertTitleElement(this.userSettingsAdd, 'Add User',
+                            '/optimized-images/user.svg');
+    this.insertTitleElement(this.adapterSettings, 'Adapters',
+                            '/optimized-image/adapters-icon.png');
```

Comment:
Missing icon?

![screen shot 2018-08-24 at 12 44 43](https://user-images.githubusercontent.com/552417/44583024-86e93500-a79b-11e8-8c72-01640e0bd70d.png)

FWIW "section title" is not very helpful alt text!

Reply:
Updated accordingly